sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -1,353 +1,356 @@
+
+Next Steps
+----------
+
+You can check that everything is installed and set up correctly by executing
+the command:
+
+.. code-block:: sh
+
+    python -m bokeh info
+
+You should see output similar to:
+
+.. code-block:: sh
+
+    Python version      :  3.8.3 | packaged by conda-forge | (default, Jun  1 2020, 17:21:09)
+    IPython version     :  7.15.0
+    Tornado version     :  6.0.4
+    Bokeh version       :  2.0.2-95-g8e0b447c0-dirty
+    BokehJS static path :  /Users/bryan/work/bokeh/bokeh/server/static
+    node.js version     :  v14.4.0
+    npm version         :  6.14.5
+
+The next check that can be made is to run some of the examples. There are
+different ways in which Bokeh can be used to suit a variety of use cases.
+
+To create an HTML file,
+
+.. code-block:: sh
+
+    BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
+
+which will create a file ``iris.html`` locally and open up a web browser.
+
+.. image:: /_images/bokeh_iris_html.png
+    :scale: 50 %
+    :align: center
+
+The variable ``BOKEH_RESOURCES`` determines where the css and JavaScript
+resources required by bokeh are found. By specifying ``inline`` we are using
+the version of BokehJS we just built to include the resources inline as part of
+the HTML file. The ``BOKEH_RESOURCES`` variable is required as the default
+behavior is to use CDN resources.
+
+Another method of running bokeh is as a server. An example of this mode of
+operation can be run using the command:
+
+.. code-block:: sh
+
+    python -m bokeh serve --show examples/app/sliders.py
+
+which will open up a browser with an interactive figure.
+
+.. image:: /_images/bokeh_app_sliders.png
+    :scale: 50 %
+    :align: center
+
+All the sliders allow interactive control of the sine wave, with each update
+redrawing the line with the new parameters. The ``--show`` option opens the
+web browser to the appropriate address, the default is ``localhost:5006``.
+
+If you have any problems with the steps here, please `contact the developers`_.
+
+.. _Anaconda: https://www.anaconda.com/distribution/
+.. _bokeh source repository: https://github.com/bokeh/bokeh
+.. _contact the developers: https://discourse.bokeh.org/c/development
+.. _conda package manager: https://docs.conda.io/projects/conda/en/latest/
+.. _Anaconda Installers: https://www.anaconda.com/products/individual
+.. _Anaconda Individual Edition: https://www.anaconda.com/products/individual
+.. _Fork a repo: https://help.github.com/en/github/getting-started-with-github/fork-a-repo
+.. _Git: https://git-scm.com
+.. _GitHub: https://github.com
+.. _GitHub Help: https://help.github.com
+.. _Installing Git: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+.. _meta.yaml: http://github.com/bokeh/bokeh/blob/master/conda.recipe/meta.yaml
+.. _Pro Git Book: https://git-scm.com/book/en/v2
+.. _this tutorial: https://www.digitalocean.com/community/tutorials/how-to-use-git-hooks-to-automate-development-and-deployment-tasks

```

Comment:
file endings again? FWIW I have vscode configured to auto-convert. We might consider adding a vscode project workspace to the top level, then it could probably configure it at that level.

Reply:
Yes, sorry - I recently reinstalled git and probably messed up my auto-convert. Should all files in this repo always be LF?

Reply:
Yes unix line endings throughout